torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2630,6 +2676,21 @@ def _make_tensor_helper(shape, low=None, high=None):
                                 dtypes=[torch.bfloat16])),
                    safe_casts_outputs=True,
                    handles_complex_extremals=False),
+    OpInfo('lerp',
+           dtypes=floating_and_complex_types(),
+           # Reference: https://github.com/pytorch/pytorch/issues/54048
+           # CUDA and ROCM don't support complex inputs
+           dtypesIfCUDA=floating_types_and(torch.half),
+           dtypesIfROCM=floating_types_and(torch.half),
+           sample_inputs_func=sample_inputs_lerp,
+           skips=(
+               # Reference: https://github.com/pytorch/pytorch/issues/53797
+               # JIT doesn't understand complex literals
+               SkipInfo('TestCommon', 'test_variant_consistency_jit',
+                        dtypes=[torch.cfloat, torch.cdouble]),
+               SkipInfo('TestOpInfo', 'test_duplicate_method_tests'),
```

Comment:
huh what does `test_duplicate_method_tests` test and why are we skipping it?

Reply:
It is to verify that operator with OpInfo does not have an entry in method_tests. We are keeping a few method tests entries where lhs is broadcasted.